IFC4x3/Properties/s/Station_2wUFXsqVDB8fwdTk7zGJXN/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Station_2wUFXsqVDB8fwdTk7zGJXN" Name="Station" UniqueId="ba78f876-d1f3-4b22-9ea7-76e1fd413857" PrimaryDataType="IfcLengthMeasure" />
```

Comment:
Did Claus not request a change to this property name? Cannot recall his proposal though.